cms/static/js/factories/login.js

```diff
@@ -22,6 +22,8 @@ define(['jquery.cookie', 'utility'], function() {
 
         $('form#login_form').submit(function(event) {
             event.preventDefault();
+            $('#submit').addClass('disabled');
+            console.log('here');
```

Comment:
@adampalay this is check so in a test when i call $("#submit").click() it should log but its not.